composer.json

```diff
@@ -32,7 +32,8 @@
 
   "require": {
     "ext-curl": "*",
-    "psr/log": "~1.0.2"
+    "psr/log": "~1.0.2",
+    "fluent/logger": "^1.0"
```

Comment:
Should we require this is a dependency even for those that do not want to use it? Maybe we should put it in `suggest` and document that one needs to ensure they have it installed in order to use this?